Possibility to specify type attribute

BilyachenkoOY opened the pull request:
Now it will be possible to specify type attribute of <style>.
This will allow to fix warning from [W3C validator](https://validator.w3.org/#validate_by_input) "The type attribute for the style element is not needed and should be omitted.".

BilyachenkoOY committed:
Possibility to specify type attr

Now it will be possible to specify or type attribute of <style>

frenzzy commented:
Why not just remove this line at all?
https://github.com/kriasoft/isomorphic-style-loader/blob/a990d947e79b47251f1af827adc81918494413fd/src/insertCss.js#L69

BilyachenkoOY commented:
For backward conpatibility at least

BilyachenkoOY commented:
@frenzzy Any news on this PR?

frenzzy commented:
@BilyachenkoOY sorry, I am not interested in maintenance `isomorphic-style-loader` anymore since I switched to using [Emotion](https://emotion.sh/).

BilyachenkoOY commented:
@koistya can you help with this? :)

dazlious reviewed (changes requested):
Just a minor change regarding the code format. Please add at least one test for this in order to allow future maintenance

dazlious committed:
Remove type attribute from style element entirely

Per W3C, the type attribute on <style> is not needed and should be
omitted, so drop it instead of making it configurable.

dazlious reviewed (approved):
Adjusted a bit to simply remove the type attribute, as I hope there is no one out there using HTML4 any longer 🤡
